### DISCUSSION TITLE
Observers in MoBi and PK-Sim

### DISCUSSION THREAD

**Discussion by sash0155** [Open-Systems-Pharmacology/Forum#1809] [Questions & Problems] [answered]
Hi,

I have created a new compartment as an observer in MoBi since the default compartments in PK-Sim are plasma, interstitial, intracellular and tissue but I needed a total tissue and unbound tissue. So assuming that tissue compartment represents the total (interstitial and intracellular), I created the unbound tissue concentration compartment by just multiplying the unbound tissue fraction with the tissue concentration in MoBi. However, when I saved the observer and import it back in PK-Sim and configure it, I couldn't see the unbound tissue compartment in the "define settings and run". Is this normal or did I miss something here?
The other option which I tried is to save the simulation from MoBi with the new compartment and import it in PK-Sim and I found a new compartment in the "define settings and run" but with this I cannot configure it since it doesn't save the building blocks.

Thanks for the help in advance.

Kind regards,
Sajid

**Comment by Yuri05** [comment]
Did you specify where the Unbound Tissue observer should be created (e.g. compare the Tissue observer below)?
If yes: create a new simulation first and check if the new observer is created where expected. Once it looks good in MoBi, import the observer again in PK-Sim.
![grafik](https://github.com/user-attachments/assets/f4b4dcfd-d602-4f95-afc3-d704380fa752)

**Reply by sash0155** [reply]
Dear @Yuri05,

Thanks for your reply.

Yes. So I sent the PK-Sim project into MoBi and then created a "Molecule Observer" and tagged it only with "Brain", since I only needed the observer in the brain as an unbound tissue. I found it in the simulations but when I save a newly created observer as a PKML and import it back in PK-SIM, I couldn't find it in the "define settings and run". One more thing is with this type of observer, MoBi name it " Brain - Intracellular" in the simulation which I don't understand since I only tagged it with Brain in the observer.
Please see the figures below;

![image](https://github.com/user-attachments/assets/0b7b62ab-be8f-46f2-a9af-69ac9db25cd6)

 
![image](https://github.com/user-attachments/assets/7ea57a8e-6163-40a8-8021-64e9d5878bf2)


Kind regards,
Sajid

**Reply by Yuri05** [reply]
Define your observer as a **Container Observer.**

Also add an additional condition `TissueOrgan` to your observer
![grafik](https://github.com/user-attachments/assets/35290db7-6c58-4ace-9c3e-a0d472c04023)

Then you should be able to find it in PK-Sim.
![grafik](https://github.com/user-attachments/assets/d4cbe5c5-c84d-4877-aa2e-03200c9c2894)

**Reply by sash0155** [reply]
Thank you @Yuri05. 
I still couldn't get the observer in PK-Sim, the reason I found is that I also introduced a new parameter under "Molecules" building block as "Fraction unbound (brain)", same like "Fraction unbound (plasma, reference value)" and then I multiplied this with Tissue concentration in the observer (which is Ok in MoBi simulations). Now, when I don't use this fraction with the new observer and import it in PK-Sim, I found this observer under "Brain" as you just showed.

I tried to define this fraction differently in MoBi (in "Parameter Start Values") but still, it's not working. Could you please let me know what would be best way to define this fraction in MoBi in order to see it in the PK-Sim observer?

![image](https://github.com/user-attachments/assets/b0456657-ce2e-4c10-8760-ece1058a1e74)

![image](https://github.com/user-attachments/assets/a8277c5a-579a-4b46-93eb-1bbdf470507b)

Kind regards,
Sajid

**Reply by Yuri05** [reply, marked as answer]
That's true - if you introduce new entities in MoBi (parameters, containers, whatsoever) - they are not exported with the observer.
At the moment, there is no possibility to pass such a parameter to PK-Sim. 
You only could decouple your observer from the new parameter and use the parameter value (or formula) **directly** in the observer formula.

**Reply by sash0155** [reply]
Thanks a lot. It worked as I just added a parameter value in the formula.
